d3a_interface/kafka_communication/kafka_producer.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import json
+from zlib import compress
+from kafka import KafkaProducer
+
+from d3a_interface.kafka_communication import KAFKA_URL, DEFAULT_KAFKA_URL, KAFKA_USERNAME, \
+    KAFKA_PASSWORD, KAFKA_COMMUNICATION_SECURITY_PROTOCOL, KAFKA_SASL_AUTH_MECHANISM, \
+    KAFKA_API_VERSION, create_kafka_new_ssl_context, KAFKA_TOPIC
```

Comment:
nit. , We can use parantheses in all 2+ lines imports

Reply:
Agreed, thanks!